src/main/java/domain/PetHotel.java

```diff
@@ -0,0 +1,46 @@
+package domain;
+
+import serenitylabs.tutorials.vetclinic.Pet;
+
+import java.util.*;
+
+import static java.util.Comparator.comparing;
+
+/**
+ * Created by pravyada on 9/20/2016.
+ */
+public class PetHotel  {
+    public static final int MAX_SIZE = 20;
+    private  Collection<Pet> petList = new TreeSet<>(comparing(Pet::getName));
+    private static Queue<Pet> petsInWaitingList = new LinkedList<>();
+
+private static final Map<HotelAvailability,CheckInStrategy> CHECK_IN_STRATEGY  = new HashMap<HotelAvailability,CheckInStrategy>();
+     {
+         CHECK_IN_STRATEGY .put(HotelAvailability.Available,new ConfirmBookingStrategy(petList));
+         CHECK_IN_STRATEGY .put(HotelAvailability.Full,new WaitingListStrategy(petsInWaitingList));
+    }
```

Comment:
Great stuff! This block could be static since the CHECK_IN_STRATEGY field is static.